java-client/barrage/src/main/java/io/deephaven/client/impl/BarrageSnapshotImpl.java

```diff
@@ -46,14 +48,17 @@
     private final ClientCallStreamObserver<FlightData> observer;
 
     private final BarrageTable resultTable;
+    private volatile Condition sealedCondition;
```

Comment:
I might put this directly above `sealed`. Fit better with that, organizationally.